src/core/library.c

```diff
@@ -889,6 +894,75 @@ QuicLibrarySetGlobalParam(
 
         break;
 
+    case QUIC_PARAM_GLOBAL_RAW_DATAPATH_PROCS: {
+        if (Buffer == NULL) {
```

Comment:
I actually think BufferLength == 0 is better now that I think of it. Theoretically they might still pass a buffer pointer, ut if they are setting zero length, I think that'd mean they are clearing this out. Thoughts?

Reply:
I think this all depends on documentation. If we document that setting a NULL buffer clears the list then we are good.

Reply:
The best API needs no documentation and just works as the caller expects it should. A caller might expect null + zero or ptr + zero, but in both cases, zero is supplied.

Reply:
Agreed.